src/app/tasks.ts

```diff
@@ -0,0 +1,5 @@
+export class Task {
+    id: number;
+    task: string;
+    status: boolean
+}
```

Comment:
the file should always end with a newline

Reply:
sure, I will follow the same.